cdap-api/src/main/java/co/cask/cdap/api/spark/SparkContext.java

```diff
@@ -211,4 +213,9 @@
    */
   @Nullable
   WorkflowToken getWorkflowToken();
+
+  /**
+   * @return the {@link TaskLocalizationContext} for the {@link Spark} program
```

Comment:
Need to document that the returned instance can be used inside Closure and that it's Serializable.